src/schema/schema.ts

```diff
@@ -64,12 +62,10 @@ export function createZimbraSchema(
 					client.getContactFrequency(variables as GetContactFrequencyOptions),
 				getConversation: (_, variables) =>
 					client.getConversation(variables as GetConversationOptions),
-				getFolder: (_: any, variables: getFolderQueryVariables) =>
```

Comment:
This was unnecessarily using enums from graphql files in the zm-x-web repo in the old version of the generated types file.